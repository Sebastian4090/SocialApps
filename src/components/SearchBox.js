import React from 'react';

const SearchBox = ({ searchChange }) =>{
    console.log('searchbox!')
    return (
        <div className='pa2'>
        <input 
            className='pa3 ba b--green bg-lightest-blue'
            type='search' 
            placeholder='search sites'
            onChange= {searchChange}/>
        </div>
    );
}

export default SearchBox;