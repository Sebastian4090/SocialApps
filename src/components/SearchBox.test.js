import { shallow } from 'enzyme';
import React from 'react';
import SearchBox from './SearchBox';

it('expect to render SearchBox component', () => {
    const mockChange = 'a';
    expect(shallow(<SearchBox searchChange={mockChange}/>)).toMatchSnapshot()
})