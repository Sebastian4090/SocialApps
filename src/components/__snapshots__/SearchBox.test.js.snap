// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render SearchBox component 1`] = `
<div
  className="pa2"
>
  <input
    className="pa3 ba b--green bg-lightest-blue"
    onChange="a"
    placeholder="search sites"
    type="search"
  />
</div>
`;
